app/assets/stylesheets/components/_header-notice.scss

```diff
@@ -21,11 +21,11 @@ $nhs_brand_color: #005EB8;
 
 .app-c-header-notice__title {
   color: govuk-colour("white");
-  margin: 0;
+  margin: govuk-spacing(0);
 }
 
 .app-c-header-notice__title-logo {
-  max-width: 90px;
```

Comment:
Personally I'd avoid the spacing scale for anything where width is concerned, I think the intent of it is only for margin and padding. Just because a number matches doesn't mean we should use the spacing scale all the time.

Reply:
I assume you meant to change this back and reduce it?

Reply:
I kind of disagree :) but I've updated it as I sort of agree with the width argument

Reply:
> I assume you meant to change this back and reduce it?

yeah I reduced it to 60px